src/components/common/Certify.vue

```diff
@@ -45,37 +45,35 @@ import { mapState } from 'vuex'
 export default class Certify extends Vue {
   readonly currentDate!: string
 
-  // Props passed into this component.
+  /** Certified By prop. */
   @Prop({ default: '' })
   private certifiedBy: string
 
+  /** Is Certified prop. */
   @Prop({ default: false })
   private isCertified: boolean
 
+  /** Message prop. */
   @Prop({ default: '' })
   private message: string
 
+  /** Entity Display prop. */
   @Prop({ default: '' })
   private entityDisplay: string
 
-  /**
-   * Called when component is created.
-   */
+  /** Called when component is created. */
```

Comment:
Don't mind this at all for one line DocBlock Comments

Reply:
Yeah, they display the same when you mouse over the property/method.